### DISCUSSION TITLE
May 18th 2021: v0.0.4-alpha

### DISCUSSION THREAD

**Discussion by rcoh** [awslabs/aws-sdk-rust#84] [General]
**New this week**:
- 🎉 Add support for AWS Lambda (smithy-rs#361, @richardhboyd) 🎉 
- Generate docs automatically and host on GitHub Pages: https://awslabs.github.io/aws-sdk-rust/ (#81)
- Add support for streaming request bodies. This is technically a **breaking change** but no currently generated AWS services expose this type. (smithy-rs#359)
- Types represented by the Smithy `Set` type now generate `Vec<T>` in all cases. This is also technically breaking but not currently exposed. (smithy-rs#270)
- Bugfix: The `.message()`field of errors will now look for both `message` and `Message` in the model (smithy-rs#374)
- Add support for the `AWS_REGION` environment variable. (smithy-rs#362)
- The request type generated by the fluent builders, eg. `dynamodb.list_tables()` is now `Debug` (smithy-rs#377, @declanvk)

And more: See the corresponding [smithy-rs release](https://github.com/awslabs/smithy-rs/releases/tag/v0.9).

Contributors:
- @richardhboyd 
- @declanvk 
- @jdisanti2019 
- @rcoh 

Thanks!

<hr /><em>This discussion was created from the release <a href='https://github.com/awslabs/aws-sdk-rust/releases/tag/v0.0.4-alpha'>May 18th 2021: v0.0.4-alpha</a>.</em>

**Comment by Doug-AWS** [comment]
Just a quick question: if one sets AWS_DEFAULT_REGION and AWS_REGION, but their values differ, who wins?

**Reply by rcoh** [reply]
Current behavior is `AWS_REGION` wins, falling back to `AWS_DEFAULT_REGION`: https://github.com/awslabs/aws-sdk-rust/blob/main/sdk/aws-types/src/region.rs#L128-L134
